psi4/src/read_options.cc

```diff
@@ -408,7 +408,8 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     to obtain the correction vector of H0BLOCK.  The ``H0BLOCK_INV``, ``GEN_DAVIDSON``,
     and ``ITER_INV`` approaches are all formally equivalent but the ``ITER_INV`` is
     less computationally expensive.  Default is ``DAVIDSON``. -*/
-    options.add_str("PRECONDITIONER", "DAVIDSON", "LANCZOS DAVIDSON GEN_DAVIDSON H0BLOCK H0BLOCK_INV ITER_INV H0BLOCK_COUPLING EVANGELISTI");
+    options.add_str("PRECONDITIONER", "DAVIDSON", "LANCZOS DAVIDSON GEN_DAVIDSON H0BLOCK ITER_INV EVANGELISTI");
+    // options.add_str("PRECONDITIONER", "DAVIDSON", "LANCZOS DAVIDSON GEN_DAVIDSON H0BLOCK H0BLOCK_INV ITER_INV H0BLOCK_COUPLING EVANGELISTI"); // Failures
```

Comment:
I'll have to look at the built docs – that read_options parser is quite good at ignoring comments and messing up the expected comment / option / comment / option pattern. Otherwise, I'd suggest removing the DETCI `CC` option from here.